### PR TITLE
updated SWOOP workflow-configuration helm chart

### PR DESCRIPTION
**Pull request by geomatician**

### CODE REVIEW

**Review by hectormachin — APPROVED** [review]

File: charts/workflow-config/templates/workflows/mirror-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                  s3:
+                    bucket: my-bucket
+                    key: "execution/{{`{{workflow.name}}`}}/input.json"
+      outputs:
+        artifacts:
+          - name: output
+            from: "{{`{{tasks.copy-assets.outputs.artifacts.output}}`}}"
+            s3:
+              bucket: my-bucket
+              key: "execution/{{`{{workflow.name}}`}}/output.json"
```
Comment (jkeifer) [inline charts/workflow-config/templates/workflows/mirror-workflow.yaml:32]: I think we switched from `execution` to `executions`?

File: charts/workflow-config/values.yaml [hunk 2]
```diff
@@ -15,8 +16,6 @@ s3:
...
     key: region
-
-minio:
-  host: minio.default
-  port: 9000
-  bucket: payloadtest
\ No newline at end of file
+  sessionTokenSecret:
+    name: copy-stac-asset-s3-secret
+    key: session_token
```
Comment (jkeifer) [inline charts/workflow-config/values.yaml:21]: No newline? Sorry, just a pet peeve of mine. 😁

File: charts/workflow-config/templates/workflows/mirror-workflow.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                  path: /mnt/input.json
+                  s3:
+                    bucket: my-bucket
+                    key: "execution/{{`{{workflow.name}}`}}/input.json"
+      outputs:
+        artifacts:
+          - name: output
+            from: "{{`{{tasks.copy-assets.outputs.artifacts.output}}`}}"
+            s3:
+              bucket: my-bucket
```
Comment (jkeifer) [inline charts/workflow-config/templates/workflows/mirror-workflow.yaml:31]: The artifact bucket name needs to be templated.

File: charts/workflow-config/templates/workflows/mirror-workflow.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  # name is required; we should never have an execution with this name
+  name: mirror-workflow
+  labels:
+    workflow-name: mirror-workflow
+spec:
+# Start of variable mappings
+{{- $artifactBucketName := .Values.s3.artifactBucketName -}}
+# End of global variable mappings
+  entrypoint: copy-assets
+  serviceAccountName: argo # set this so the proper permissions are assigned
```
Comment (jkeifer) [inline charts/workflow-config/templates/workflows/mirror-workflow.yaml:13]: This should be a chart input

**Review by hectormachin — APPROVED** [review]

**Pull request merged**